db/migrate/20200420171112_create_patients.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePatients < ActiveRecord::Migration[6.0]
+  def change
+    create_table :patients do |t|
+      t.string "first_name"
+      t.string "last_name"
+      t.string "email"
+      t.integer "phone"
+      t.timestamps
```

Comment:
We have a style of adding a blank line above `t.timestamps` - another one to add to iomt-rubocop